applications/StructuralMechanicsApplication/custom_response_functions/response_utilities/adjoint_postprocess.cpp

```diff
@@ -156,6 +167,10 @@ namespace Kratos
         {
             // overwrite existing.
             this->SetAllSensitivityVariablesToZero();
+        }        
+        else if (mBuildMode == "static-non-linear")
```

Comment:
but what is with e.g. displacement response in a non-linear situation? This build mode behavior is not suitable for all responses within a static non-linear analysis.

Reply:
I can change it so it is specifically for strain energy nonlinear static